markdown/dt-otel/dt-otel.md

```diff
@@ -174,31 +244,183 @@ solbroker(configure)# message-vpn default
+  "rejectMsgToSenderOnNoSubscriptionMatchEnabled": true
+}'
+```
+### CLI
+
 ```console
 solbroker# configure
 solbroker(configure)# client-profile default message-vpn default
 solbroker(configure/client-profile)# message-spool reject-msg-to-sender-on-no-subscription-match
 solbroker(configure/client-profile)# end
 ```
 
-### Configuring the Telemetry Profile
+## Telemetry Profile Configuration
 
 The Telemetry Profile defines which published messages should be traced as well as who should be allowed to consume those trace messages.
 
+When creating a Telemetry Profile, a Telemetry Queue is created. In this example, the queue name would be `#telemetry-trace` because we use `trace` as the profile name when creating the Telemetry Profile. 
+When generated, trace messages will be added to this queue for consumption.
+
+Creating a Telemetry Profile will also cause the broker to create a Client Profile as well as an ACL Profile. Just like the Telemetry Queue, the names of these profiles will take on the format of `#telemetry-&#60;telemetry-profile-name&#62;`.
+These profiles must be used by the Client Username or else the Client will not be able to bind to the Telemetry Queue to consume trace messages.
+
+Below is a snippet from the Open Telemetry Collector configuration included in the tracing-codelab downloaded earlier. Notice how the username, password, and queue name all match the settings configured on the broker.
+Be sure to update the collector configuration should any of the Telemetry Profile config change on the broker.
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+
+  solace:
+    broker: [solbroker:5672]
+    max_unacknowledged: 500
+    auth:
+      sasl_plain:
+        username: trace
+        password: trace
+    queue: queue://#telemetry-trace
+    tls:
+      insecure: true
+      insecure_skip_verify: true
+```
+
+First, start by creating the Telemetry Profile.
+
+### PubSub+ Manager
+
+Within the default message VPN, navigate to Telemetry and select _Create Telemetry Profile_
+
+![alt-text-here](img/pubsub-manager-7.png)
+
+Name the profile _trace_ and click Apply. We'll come back to this page later to update additional settings.
+
+![alt-text-here](img/pubsub-manager-8.png)
+
+Next, we need to enable the receiver. From the trace Telemetry Profile page, select _Receiver Connect ACLs_ and update the _Client Connect Default Action_ to _Allow_
+  _Tip: Double-click the input to enable edit mode_
+
+![alt-text-here](img/pubsub-manager-9.png)
+
+![alt-text-here](img/pubsub-manager-10.png)
+
+After applying the ACL, edit the trace Telemetry Profile page to enable the _Reciever_ and _Trace_ settings.
+
+![alt-text-here](img/pubsub-manager-11.png)
+
+Finally, let's create a Trace Filter and add a subscription that will attract all topic messages (using the `>` subscription)
+
+Create the filter with name _default_. Be sure to enable before clicking Apply.
+![alt-text-here](img/pubsub-manager-12.png)
+
+Add the '>' subscription
```

Comment:
To stay consistent with code formatting, I would add back ticks ` instead of single quote to have a `>` code formatting (instead of '>')